### PR TITLE
[Fix] Remove `fire` from kannon's dependencies

### PR DESCRIPTION
**Pull request by maronuu**
## What
Addressing issue #21.

### CODE REVIEW

File: poetry.lock [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.4.1 and should not be changed by hand.
```
Comment (yokomotod) [inline poetry.lock:1]: Probably not a problem, but I don't want to lower the version.

File: example/requirements.txt [hunk 2]
```diff
@@ -1 +1,2 @@
+fire
```
Comment (yokomotod) [inline example/requirements.txt:1]: [Out of this PR scope] 

I think now we should use the Poetry and control the outside library's version.

I made #28

**Review by yokomotod — CHANGES_REQUESTED** [review]
@maronuu Thank you for your work! Almost fine but can you re-lock with latest poetry?

File: poetry.lock (hunk 1 above)
Comment (maronuu) [reply poetry.lock:1]: re-locked with the latest poetry 1.4.2 in [47d27fd](https://github.com/m3dev/kannon/pull/27/commits/47d27fdee775c5e9ea08cef98a42b1305f09f4d9)

**Review by kitagry — APPROVED** [review]
Thank you!

**Review by yokomotod — APPROVED** [review]
thx!

**Pull request merged**